### PR TITLE
updates mermaid diagram

### PR DESCRIPTION
**Pull request by dawnkelly09**
I took the gray background out but this is still pretty small. See what you think. I was happy with the rest.

### CODE REVIEW

**Comment by albertov19** [comment]
We could make it so that the first sugraph is left to right in the first three boxes and then vertical for the 3-4 and right to left from 4-5 and then there is vertical arrows connecting subgraphs 1 -> 2  -> 3

**Review by eshaben — COMMENTED** [review]
Yeah give Alberto's suggestion a try and see what that looks like. Thanks for checking these all out

**Review by eshaben — COMMENTED** [review]
It's not rendering correctly 

Screenshot 2024-11-23 at 1 22 05 PM

File: develop/blockchains/get-started/deploy-blockchain-to-polkadot.md [hunk 1]
```diff
@@ -16,19 +16,43 @@ Whether deploying a test network for development or a mainnet for production, th
...
+        space:2 id1(["Pre-Deployment"]):1 space:2
+        A("Local \nDevelopment \nand Testing")
+        space
+        B("Runtime \nCompilation")
+        space
+        C("Generate \nChain \nSpecifications")
+        space:7
+        E("Acquire \nCoretime")
+        space
+        D("Prepare \nDeployment \nEnvironment")
```
Comment (eshaben) [inline develop/blockchains/get-started/deploy-blockchain-to-polkadot.md:32]:
```suggestion
flowchart TD
    %% Group 1: Pre-Deployment
    subgraph group1 [Pre-Deployment]
        direction LR
        A("Local \nDevelopment \nand Testing") --> B("Runtime \nCompilation")
        B --> C("Generate \nChain \nSpecifications")
        C --> D("Prepare \nDeployment \nEnvironment")
        D --> E("Acquire \nCoretime")
```

File: develop/blockchains/get-started/deploy-blockchain-to-polkadot.md [hunk 2]
```diff
@@ -16,19 +16,43 @@ Whether deploying a test network for development or a mainnet for production, th
...
-    E --> F("Launch \n and \n Monitor")
+        A-->B 
+        B-->C
+        C-->D 
+        D-->E
+    block:group2:1
+        columns 1
+        id2(["Deployment"]) 
+        space
+        F("Launch \nand \nMonitor")
```
Comment (eshaben) [inline develop/blockchains/get-started/deploy-blockchain-to-polkadot.md:42]:
```suggestion
    
    %% Group 2: Deployment
    subgraph group2 [Deployment]
        F("Launch \nand \nMonitor")
```

File: develop/blockchains/get-started/deploy-blockchain-to-polkadot.md [hunk 3]
```diff
@@ -16,19 +16,43 @@ Whether deploying a test network for development or a mainnet for production, th
...
     end
-    subgraph Post-Deployment
-    F --> G("Maintenance \n and \n Upgrades")
+    block:group3:1
+        columns 1
+        id3(["Post-Deployment"])
+        space
+        G("Maintenance \nand \nUpgrades")
```
Comment (eshaben) [inline develop/blockchains/get-started/deploy-blockchain-to-polkadot.md:48]:
```suggestion

    %% Group 3: Post-Deployment
    subgraph group3 [Post-Deployment]
        G("Maintenance \nand \nUpgrades")
```

File: develop/blockchains/get-started/deploy-blockchain-to-polkadot.md [hunk 4]
```diff
@@ -16,19 +16,43 @@ Whether deploying a test network for development or a mainnet for production, th
...
     end
+        id1-->id2
+        id2-->id3
+    %% Style adjustments
+    style group1 fill:#ffffff,stroke:#6e7391,stroke-width:1px
+    style group2 fill:#ffffff,stroke:#6e7391,stroke-width:1px
+    style group3 fill:#ffffff,stroke:#6e7391,stroke-width:1px
```
Comment (eshaben) [inline develop/blockchains/get-started/deploy-blockchain-to-polkadot.md:55]:
```suggestion

    %% Connections Between Groups
    group1 --> group2
    group2 --> group3

    %% Styling
    style group1 fill:#ffffff,stroke:#6e7391,stroke-width:1px
    style group2 fill:#ffffff,stroke:#6e7391,stroke-width:1px
    style group3 fill:#ffffff,stroke:#6e7391,stroke-width:1px
```

**Comment by eshaben** [comment]
It wouldn't allow me to suggest the changes at once. So I had to break it up. But this is what the entire diagram code should be:

```
flowchart TD
    %% Group 1: Pre-Deployment
    subgraph group1 [Pre-Deployment]
        direction LR
        A("Local \nDevelopment \nand Testing") --> B("Runtime \nCompilation")
        B --> C("Generate \nChain \nSpecifications")
        C --> D("Prepare \nDeployment \nEnvironment")
        D --> E("Acquire \nCoretime")
    end
    
    %% Group 2: Deployment
    subgraph group2 [Deployment]
        F("Launch \nand \nMonitor")
    end

    %% Group 3: Post-Deployment
    subgraph group3 [Post-Deployment]
        G("Maintenance \nand \nUpgrades")
    end

    %% Connections Between Groups
    group1 --> group2
    group2 --> group3

    %% Styling
    style group1 fill:#ffffff,stroke:#6e7391,stroke-width:1px
    style group2 fill:#ffffff,stroke:#6e7391,stroke-width:1px
    style group3 fill:#ffffff,stroke:#6e7391,stroke-width:1px
```

**Review by albertov19 — APPROVED** [review]
LGTM!

I've been using `<br/>` instead of ` \n` in my mermaids.  Mainly because ` \n` does not work with https://mermaid.live/

**Comment by albertov19** [comment]
I do see this warning btw `[23:20:14] "GET /tutorials/polkadot-sdk/testing/spawn-basic-chain/ HTTP/1.1" code 404`

**Comment by eshaben** [comment]
I have to imagine that warning is because you have a tab open on that page, but that page has moved

**Pull request merged**